fbfmaproom/environment_linux.yml

```diff
@@ -1,44 +1,47 @@
-name: fbfmaproom2
+name: dashtest
```

Comment:
Oops, I unwittingly changed the name of the environment here, and the container build depended on it. I fixed that directly on master--now we use `conda env create -n app` so this line is ignored.